store_map.py

```diff
@@ -0,0 +1,58 @@
+ad_name2index = {
```

Comment:
Here again you can hard-coding information, which is 1) not very pretty and 2) not reliable because the adventures can change over time. Can you grab this from the yamls?